### PR TITLE
 Fixed datetime, fixed readonly, fixed operatorsMap 

### PR DESCRIPTION
**Pull request by mpf82**
Fixed datetime search in grid,
introduced w2utils.settings.datetimeFormat,
accessing readonly / disabled is now done with prop() instead of attr(),
fixed a bug where you could select items from a disabled list - see
issue #1153,
operatorsMap for date, time and datetime now points to "date" instead of
"number"

### CODE REVIEW

File: src/w2utils.js [hunk 1]
```diff
@@ -247,7 +248,7 @@ var w2utils = (function ($) {
     }
 
     function isDateTime (val, format, retDate) {
-        if (format == null) format = w2utils.settings.dateFormat + '|' + w2utils.settings.timeFormat;
+        if (format == null) format = w2utils.settings.datetimeFormat;
```
Comment (mrg2001) [inline src/w2utils.js]: Why not use dateFormat + timeFormat? Less definitions.

Comment (mpf82) [inline src/w2utils.js]: Users might want to have a different format for date, time and timetime. for example they could want to show seconds on time, but not on datetime.
Or the other way around, they might need a higher precision on datetime, but not on time.

Comment (mrg2001) [inline src/w2utils.js]: Sure, but it is default format. There is always a way to define custom for any particular case. Anyway, I think it is not a big issue. The pull request is merged. Thank you for your contribution.

Comment (mpf82) [inline src/w2utils.js]: You're welcome.

Could you please regenerate the files in the dist folder?

**Pull request merged**